receiver/dotnetdiagnosticsreceiver/dotnet/request.go

```diff
@@ -140,14 +140,14 @@ func createProviders(intervalSec int, names ...string) (ps []provider) {
 }
 
 // from https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventlevel
-const verboseEventLevel = 5
+const verboseEventLevel = 2
```

Comment:
Is this change needed?

Reply:
It's not necessary. 
The dotnet-counters source code is `EventLevel.Error`. https://github.com/dotnet/diagnostics/blob/main/src/Tools/dotnet-counters/CounterMonitor.cs#L774